### DISCUSSION TITLE
Files changed coverage rate always shows up as n/a

### DISCUSSION THREAD

**Discussion by balaji-srin** [zgosalvez/github-actions-report-lcov#34] [Q&A]
Has anyone gotten the "Files changed coverage rate" to show some number? It always shows n/a to me even if I introduce a change to a file that is covered by unit tests.

**Comment by balaji-srin** [comment]
@zgosalvez Any idea?

**Reply by mccrearyp** [reply]
Same happens to me, never seen it work in my repo
